modular_skyrat/modules/modular_implants/code/nifsoft_catalog.dm

```diff
@@ -0,0 +1,99 @@
+GLOBAL_LIST_INIT(available_nifsofts, list(
```

Comment:
I feel like it should be more descriptive of what they're available for. Available to use? To purchase? To download?

Being a bit more specific is important for global variables.